.github/workflows/cookieplone.yml

```
@@ -18,6 +18,13 @@ jobs:
       project-directory: volto-add-on
     steps:
       - uses: actions/checkout@v4
+
+      - name: Compute several vars needed for the CI
+        id: vars
+        run: |
+          python3 -c 'import json; data = json.load(open("./packages/volto/package.json")); print("volto-version=" + (data.get("version") or "19.0.0-alpha.0"))' >> $GITHUB_OUTPUT
```

I think it's better to give an error if the version is not found, as it can mask errors and the fallback may become outdated.

```suggestion
          python3 -c 'import json; data = json.load(open("./packages/volto/package.json")); print("volto-version=" + data["version"])' >> $GITHUB_OUTPUT
```

---

Another option would be to move this step to after Node configuration and do it with JavaScript. It's simpler, and we can accommodate developers who don't understand Python:

```bash
node -e "const v = require('./packages/volto/package.json').version || process.exit(1); console.log('volto-version=' + v)" >> $GITHUB_OUTPUT
```